content/documentation/staging/features/index.adoc

```diff
@@ -816,12 +816,39 @@ Regarding the *severity* of each component, there are only to options: `core` or
 
 By default, Kiali checks the following components installed in the control plane namespace: istiod, ingress, egress; and prometheus, grafana and jaeger accessible thought their services.
 
-== Multi-cluster support
-Istio provides https://istio.io/docs/setup/install/multicluster/[installation instructions] for three different multi-cluster scenarios: replicated control planes, shared control plane with single-network, and shared control plane with multi-network.
+== Advanced Mesh and Multi-cluster support
+A basic Istio mesh deployment has a single control plane with a single data plane, deployed on a single Kubernetes cluster. But Istio supports a variety of advanced https://istio.io/latest/docs/ops/deployment/deployment-models/[deployment models, window="_blank"]. It allows a mesh to span multiple primary (control plane) and/or remote (data plane only) clusters, as well as multiple networks.  The only strict rule is that within a mesh service names are unique. A non-basic mesh deployment generally involves multiple clusters.  See https://istio.io/docs/setup/install/multicluster/[installation instructions, window="_blank"] for more detail on installing advanced mesh deployments.
```

Comment:
I read `as well as multiple networks`, which may be a little ambiguous. The _"multiple networks"_ refers to the pod networks.

It's more a kubernetes+network config detail rather than Istio detail, but Istio is taking it into account to let you choose how to manage traffic.

If two clusters have it's own "pod network", a pod in one cluster can have the same IP address to a pod on the other cluster and all is going to work fine, because networks are isolated. Since networks are isolated, we must use Istio gateways to enable communication between both clusters.

However, if two clusters have a "flat pod network" there must be no IP address clashes. The "flat pod network" has the benefit (or disadvantage?) that pods in one cluster can directly communicate with pods on the other cluster and we no longer need the Istio gateways.

I think it's important to remove ambiguity, because people may think it's the network where clusters are located (i.e. they may think this is for separate data centers).

Not sure how we can denote all this in a very short sentence to remove ambiguity :-) This is already explained in Istio docs [here](https://istio.io/latest/docs/ops/deployment/deployment-models/#multiple-networks), so I think we shouldn't explain that again. Perhaps somehow remove the ambiguity and redirect people to the istio docs.